warrior/WarriorCore/project_driver.py

```diff
@@ -197,203 +187,30 @@ def execute_project(project_filepath, auto_defects, jiraproj, res_startdir, logs
-                                  "the data repository".format(retry_cond_value))
-
-                    condition_met = False
-                if condition_met is False:
-                    goto_testsuite = str(retry_value)
-            else:
-                if retry_type.upper() == 'IF NOT':
-                    try:
-                        if data_repository[retry_cond] != retry_cond_value:
-                            condition_met = True
-                            pNote("Wait for {0}sec before "
-                                  "retrying".format(retry_interval))
-                            pNote("The condition value '{0}' does not match "
-                                  "with the expected value '{1}'".
-                                  format(data_repository[retry_cond],
-                                         retry_cond_value))
-                            time.sleep(int(retry_interval))
-                        else:
-                            condition_met = False
-                    except KeyError:
-                        condition_met = False
-                        print_warning("The given condition '{0}' is not there "
-                                      "in the data repository".format(retry_cond_value))
-                    if condition_met is False:
-                        pNote("The given condition '{0}' matched with the "
-                              "value '{1}'".format(data_repository[retry_cond],
-                                                   retry_cond_value))
-                        goto_testsuite = str(retry_value)
-        else:
-            if testsuite_status is False or testsuite_status == "ERROR" or\
-                    testsuite_status == "EXCEPTION":
-                goto_testsuite = onerror_driver.main(testsuite, project_error_action,
-                                                     project_error_value)
-            if goto_testsuite in ['ABORT', 'ABORT_AS_ERROR']:
-                break
-            # when 'onError:goto' value is less than the current ts num,
-            # change the next iteration point to goto value
-            elif goto_testsuite and int(goto_testsuite) < suite_cntr:
-                suite_cntr = int(goto_testsuite)-1
-                goto_testsuite = False
-
-    project_status = Utils.testcase_Utils.compute_status_using_impact(ts_status_list,
-                                                                      ts_impact_list)
+    data_repository["war_parallel"] = False
+
+    execution_type = Utils.xml_Utils.getChildAttributebyParentTag(project_filepath, 'Details',
+                                                                  'type', 'exectype')
+
+    # for backward compatibility(when exectype is not provided)
+    if execution_type is False:
+        execution_type = "sequential_suites"
+
+    if execution_type.upper() == 'PARALLEL_SUITES':
+        pj_junit_object.remove_html_obj()
+        data_repository["war_parallel"] = True
```

Comment:
can we make the attribute ts_parallel instead of war_parallel to indicate suites are running in parallel? Do we have similar attribute tc_parallel for testcases running in parallel?